src/schemas/base-field.ts

```diff
@@ -0,0 +1,8 @@
+import { EFieldType } from "./types";
+
+export type IBaseField = {
+    id: string;
+    internalId: string;
+    label: string;
+    type: EFieldType;
+};
```

Comment:
this should be an interface right? other field types can then extend from `IBaseField`